README.md

```diff
@@ -86,6 +86,33 @@ Note: In this configuration, you must do some additional configuration:
 - If you wish your Plex Media Server to be accessible outside of your home network, you must manually setup port forwarding on your router to forward to the `ADVERTISE_IP` specified above.  By default you can forward port 32400, but if you choose to use a different external port, be sure you configure this in Plex Media Server's `Remote Access` settings.  With this type of docker networking, the Plex Media Server is essentially behind two routers and it cannot automatically setup port forwarding on its own.
 - (Plex Pass only) After the server has been set up, you should configure the `LAN Networks` preference to contain the network of your LAN.  This instructs the Plex Media Server to treat these IP addresses as part of your LAN when applying bandwidth controls.  The syntax is the same as the `ALLOWED_NETWORKS` below.  For example `192.168.1.0/24,172.16.0.0/16` will allow access to the entire `192.168.1.x` range and the `172.16.x.x` range.
 
+### ARM notes
+
+To use the docker-compose scripts with ARM (SBCs like Raspberry Pi), you must first build the docker image locally.
+
+```sh
+docker build -f Dockerfile.armv7 # or arm64
+```
+
+Then, remove the `image: plexinc/pms-docker` line from your chosen compose template, and add the line `build: .` in its place.
```

Comment:
If you build with `-t plexinc/pms-docker:latest` then this (and the below) is completely unnecessary.

Reply:
I'm not sure what you mean. I was not able to get a working build with the standard image because it's not ARM compatible. Can you elaborate more?

Reply:
For additional clarification, this is not about getting a "latest build" this is about building the **arm64/armv7** image (which are not provided though Docker Hub), just like you have to for ARM devices. The additional context here is how to do that *in conjunction with* using the provided `docker-compose` templates.

I can reword my changes to make that more obvious.

Reply:
I know exactly what you are trying to do; I'm telling you there's a better way:  When you run `docker build`, you can specify the tag under which it'll store the resulting image.  So, use the one expected by docker-compose in the existing template.  Then all you need is the build command.

Reply:
Sorry, it seemed like you were saying not to build it yourself, I misread.

I'll update the instruction, thanks.